Sources/PackageModel/Sources.swift

```diff
@@ -29,7 +29,7 @@ public struct Sources {
     
     static public var validSwiftExtensions = Set<String>(["swift"])
     
-    static public var validCExtensions = Set<String>(["c"])
+    static public var validCExtensions = Set<String>(["c", "m", "cc", "cpp", "cxx"])
```

Comment:
While we are doing this, is there any reason not to include Objective-C++ at the same time? (`.mm`)